README.md

```diff
@@ -1,2 +1,131 @@
 # Portfolio_Microverse
-A responsive website project that describe my achievements.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+- [📝 License](#license)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio_Microverse] <a name="about-project"></a>
+
+
+**[Portfolio_Microverse]** Is a responsive website project that describe my achievements.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+## Html
+## Css
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Present myself as a software developer]**
+<!-- - **[My Skills]** -->
+- **[My contact]**
+<!-- - **[My past projects]**
+- **[Responsiveness ]**
+- **[Desktop and Mobile version]**
+- **[Interactive hamburger menu]**
+- **[Modal pop up on clicking to see project]** -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Figma Link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=teTWva4DHx9UzJUX-0)
+
+[Live Demo Link](https://Mukaba/Portfolio_Microverse/)
```

Comment:
- [ ] Kindly remove the `Live Demo Link` as currently your project is not deployed yet, you can simply replace that with a text stating `Not available`